packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -59,6 +59,14 @@ export default class SelectBox extends PureComponent {
          */
         onValueChange: PropTypes.func.isRequired,
 
+        /**
+         * This prop gets called when an option was selected,
+         * the inspector is not in dirty status and
+         * the NodeCreationDialog is not open.
+         * It handles the linking of the reference editor.
+         */
```

Comment:
```suggestion
        /**
         * This prop gets called when the select box header element gets clicked.
         */
```
This is to avoid merge conflicts with: https://github.com/neos/neos-ui/pull/3341